packages/hash/frontend/src/pages/[account-slug]/index.page.tsx

```diff
@@ -8,10 +8,14 @@ import { Link } from "../../shared/ui";
 import { useRouteAccountInfo } from "../../shared/routing";
 
 const Page: NextPageWithLayout = () => {
-  const { authenticatedUser } = useAuthenticatedUser();
+  const { loading, authenticatedUser } = useAuthenticatedUser();
   const { orgs } = useOrgs();
   const { accountId } = useRouteAccountInfo();
 
+  if (loading) {
+    return null;
+  }
```

Comment:
This was previously flashing up the "You must be logged in..." message when the session was loading.